src/nupic/types/Sdr.cpp

```diff
@@ -407,6 +407,7 @@ namespace nupic {
         // Consume the end marker.
         inStream >> marker;
         NTA_CHECK( marker == "~SDR" );
+        inStream.ignore(1);  // skip past endl.
```

Comment:
maybe not, so we'd detect broken stream, saves?

Reply:
I had to add this because without it my load( ) was broken. :)

Reply:
+1 This is good.  Method `SDR.save()` always appends a newline to the stream which this correctly chomps off.